siddpubs-conf.bib

```diff
@@ -63,6 +63,13 @@ @string{ldc
 
 % 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{ke2021grasping,
+ title = {{Grasping with Chopsticks: Combating Covariate Shift in Model-free Imitation Learning for Fine Manipulation}},
+ author = {Ke, L., Wang, J., Bhattacharjee, T., Boots, B. and Srinivasa, S.S.},
```

Comment:
I think the website assumes that authors are separated by ` and `, see all other entries below.

Best to keep that consistent